### ISSUE TITLE
SVG elements on webbook lack alt text 

### ISSUE THREAD

**Issue by mcgratay** [pressbooks/pressbooks-book#617]
### Description

A few SVG elements included by default on all webbooks lack titles. These have been brought to our attention from Pressbooks users. 

- the Pressbooks logo in the bottom left corner of the footer 
- the previous and next arrows that help you navigate from chapter to chapter

![Screen Shot 2019-10-30 at 2 16 38 PM](https://user-images.githubusercontent.com/39064028/67886481-e4a3aa80-fb1f-11e9-9936-157c1bd6ad42.png)

### Steps to Reproduce

1. Visit a webbook 
2. Choose a chapter
3. Scroll to the bottom of the page
4. User a screen reader to navigate to the navigation cues and the Pressbooks logo (which links to Pressbooks.com) 

**Expected behavior:** All visual elements are announced 

**Actual behavior:** These specific SVGs do not yet have a title which would make them accessible to screenreaders.

**Comment by SteelWagstaff** [comment]
See method used in https://github.com/pressbooks/pressbooks-book/pull/608 for model

**Comment by SteelWagstaff** [comment]
Opened PRs in Aldine and Pressbooks-book to add titles to SVG elements in footer and reading interface.

**Comment by arzola** [comment]
I reviewed your PRs Steel, I feel those should have to be translatable strings

**Comment by fdalcin** [comment]
I’d like to discuss this a bit further. Alt text should be used when an image provides meaningful information or context. However, if the image is purely decorative or used for visual presentation (e.g., spacers, borders, or non-informative illustrations), then we should set `alt=""` so screen readers skip it.

My understanding is that alt text should focus on describing the function or meaning of the image in its current context. Adding generic labels like `whatever logo` can actually hinder the user experience for screen reader users, especially if the content is already described in surrounding text.

For example, what I understand is that the `LinkedIn Logo` message here does not add much context since the link already tells users the context

```html
<a href="...">
    <img src="..." alt="LinkedIn Logo" />
    <span class="screen-reader-text">Pressbooks on LinkedIn</span>
</a>
```

**Comment by cmurtagh** [comment]
I would like to have @greatislander's thoughts on this, as I think Dalcin raises a fair point.

**Comment by greatislander** [comment]
> I’d like to discuss this a bit further. Alt text should be used when an image provides meaningful information or context. However, if the image is purely decorative or used for visual presentation (e.g., spacers, borders, or non-informative illustrations), then we should set `alt=""` so screen readers skip it.
> 
> My understanding is that alt text should focus on describing the function or meaning of the image in its current context. Adding generic labels like `whatever logo` can actually hinder the user experience for screen reader users, especially if the content is already described in surrounding text.
> 
> For example, what I understand is that the `LinkedIn Logo` message here does not add much context since the link already tells users the context
> 
> <a href="...">
>     <img src="..." alt="LinkedIn Logo" />
>     <span class="screen-reader-text">Pressbooks on LinkedIn</span>
> </a>

You are correct in this instance — generally if a logo is part of a link that already has an accessible name you should not include alt text for it. `alt=""` and `aria-hidden="true"` are both often used in this situation. I'd need to look at these on a case by case basis though as there may be other cases where the SVG should have an accessible name.

**Comment by richard015ar** [comment]
We want to discuss this further with Ned. We might request a meeting considering @cmurtagh also have other topics he might want to chat about.

**Comment by greatislander** [comment]
Shared this with @cmurtagh just now, it's a good reference for how to decide whether or not to give an icon an accessible name: https://docs.fontawesome.com/web/dig-deeper/accessibility

**Comment by SteelWagstaff** [comment]
@arzola The PRs addressing this have all been merged and are included in the newly released versions of Aldine and McLuhan. I think we can close this as done now?
